Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -101,26 +95,26 @@ @interface OWSProfileManager ()
 @property (nonatomic, readonly) YapDatabaseConnection *dbConnection;
 @property (nonatomic, readonly) TSNetworkManager *networkManager;
 
+// These properties can be accessed on any thread, while synchronized on self.
```

Comment:
Yup. I've tried to make things clearer by adopting a single policy: "only access the mutable state while synchronized on self".  Anyway, I've added per-property comments, but they're all the same now.